docs/file-formats/xml-data-file.md

```diff
@@ -20,11 +20,11 @@ The import file must start with the standard XML header:
 
 The `<knora>` element describes all resources that should be imported. It has the following attributes:
 
```

Comment:
here I fixed the line-length violation, and in the same shot, make the text more consistent